mountpoint-s3/src/inode.rs

```diff
@@ -1069,7 +1069,10 @@ impl WriteHandle {
                 })
             }
             WriteStatus::LocalOpen => Err(InodeError::InodeAlreadyWriting(inode.err())),
-            WriteStatus::Remote => Err(InodeError::InodeNotWritable(inode.err())),
+            WriteStatus::Remote => Err(InodeError::InodeNotWritable(
+                inode.err(),
+                "Writing to existing file is NOT supported by Mountpoint.".to_owned(),
```

Comment:
Rather than adding a message here, I'd prefer having a specific `InodeError` case for each situation.